invoice/pdf_generator.py

```diff
@@ -58,9 +58,11 @@ def render_address(x, y, address, prefix_lines=(), suffix_lines=(), line_height=
     # Customer address
     render_address(A4_WIDTH - 200, y_top, invoice.customer.address, prefix_lines=[invoice.customer.full_name])
 
-    # Title
+    # Title, number, date
     y_top = A4_HEIGHT - 200
     pdf_object.drawString(x_left, next_y(), "Invoice")
```

Comment:
I think we should make this bold as it looks like "Invoice Number" has a line break.
![image](https://github.com/user-attachments/assets/fc4dfd71-2d35-4e7e-99e1-17528b03c0e9)